dspace-services/src/main/java/org/dspace/services/sessions/StatelessRequestServiceImpl.java

```diff
@@ -102,7 +103,7 @@ private String startRequest(Request req) {
         for (RequestInterceptor requestInterceptor : interceptors) {
             if (requestInterceptor != null) {
                 try {
-                    requestInterceptor.onStart(req.getRequestId(), req.getSession());
+                    requestInterceptor.onStart(req.getRequestId(), null);
```

Comment:
as the concept of session has been completely withdrawn (the SessionService interface was deleted) it should be better to change the method signature